{"classFile":"CircuitBreakingExceptionTest","className":"CircuitBreakingExceptionTest","packages":"org.apache.commons.lang3.concurrent","methodInfos":[{"md5":"bqEW+B8arV0hBX0k1Isj2w==","methodName":"testThrowingInformativeException","parameters":"[]"},{"md5":"prO7ZEH5ikdQHQm0jDdBCQ==","methodName":"testThrowingExceptionWithMessage","parameters":"[]"},{"md5":"GjUZScDwiwKs0skdB3yFqg==","methodName":"testThrowingExceptionWithCause","parameters":"[]"},{"md5":"fVxip/kU2hSMIczcz4Gbag==","methodName":"testThrowingEmptyException","parameters":"[]"},{"md5":"TxeED58b2f+44J99UiUYVA==","methodName":"testWithCauseAndMessage","parameters":"[]"},{"md5":"wwn2aiNumpkSk0ZYu4wb8g==","methodName":"testWithoutCause","parameters":"[]"},{"md5":"16I9EVprYnV3cm0s+TLUJA==","methodName":"testWithoutMessage","parameters":"[]"}],"addLines":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,80,81,82,83],"delLines":[],"type":"ADD","path":"src/test/java/org/apache/commons/lang3/concurrent/CircuitBreakingExceptionTest.java","pro_Commit":"aadb9a31ed969fc72cc87d486ec02383faa5bd06","label":null,"predict":null,"commitID":"e8d861359ade336badcac3211e5aa447097479a6","pro_Date":"2016-10-24 01:55:14","file_Date":"2017-01-06 03:53:01","projectName":"commons-lang","sha1":"cdcd679be8610f6a822128716b5e0ef098a7d146","pro_Path":null}